app/controllers/api/v1/greeting_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::GreetingController < ApplicationController
+  def index
+    @greetings = Greeting.all
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, according to the requirements:

![image](https://github.com/haliljon/hello-rails-back-end/assets/77069011/693212ed-1da7-4239-8bb2-aa42237767a1)

This endpoint should return one random Greeting and not all greetings. Please consider fixing this! 🤓 🔧

Reply:
Hi @anagudelogu 

It is done ✅